app/managers.py

```diff
@@ -0,0 +1,44 @@
+import sqlite3
+
+from app.models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self.__connection = sqlite3.connect("cinema.db3")
+        self.tabl_name = "actors"
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self.__connection.execute(
+            f"INSERT INTO {self.tabl_name} "
+            "(first_name, last_name) VALUES (?, ?)",
+            (first_name, last_name)
+        )
+        self.__connection.commit()
+
+    def all(self) -> list:
```

Comment:
You could indicate a list of what instance is returned in this method in the return type annotation.

Reply:
Not fixed. You should indicate that you expect to obtain `list[Actor]`.